src/test/java/gov/healthit/chpl/aqa/stepDefinitions/Base.java

```diff
@@ -162,5 +164,24 @@ public String getCurrentDate() {
         String dateFormatted = dateFormat.format(date);
         return dateFormatted;
     }
+
+    /**
+     * Get Date with Time Zone.
+     * @param date in MMMM dd,yyyy with time stamp and GMT time zone
+     * @throws ParseException if unable to parse the date
+     * @return date
+     */
+    public Date getFormatedDateTimeZone(final String date) throws ParseException {
+        Date dateInstance;
+        DateFormat dateFormat = new SimpleDateFormat("MMMM dd,yyyy hh:mm:ss a z");
+        dateFormat.setTimeZone(TimeZone.getTimeZone("GMT"));
+        if (date == null) {
+            String currentDate = dateFormat.format(new Date());
```

Comment:
```suggestion
            return new Date();
```

There's no reason to create a string using a new Date only to then parse it into a Date object. Just return the new Date.

Should also probably move this entire check to the first line of the method. No reason to do any other work if the Date doesn't exist in the first place